resources/web/wwi/nodes/utils/math_utils.js

```diff
@@ -1,9 +1,25 @@
-function clampedAcos(value) {
-  if (value >= 1.0)
-    return 0.0;
-  if (value <= -1.0)
-    return 2.0 * Math.PI;
+export function clampedAcos(value) {
```

Comment:
just a detail, but this file is called `math_utils` whereas for the node we call it `node_utilities`, we should pick one of the two